config/initializers/active_admin.rb

```diff
@@ -6,7 +6,7 @@
   # Set the title that is displayed on the main layout
   # for each of the active admin pages.
   #
-  config.site_title = 'App'
+  config.site_title = 'App Admin'
```

Comment:
Should this be modified as well? I think you are only searching for "App", right?

Reply:
@JulianPasquale  According to the ticket, the idea would be to add `Admin` after the `app-name`, which is the reason why I added it here, thoughts?

https://github.com/rootstrap/rails_api_base/issues/318#:~:text=It%20could%20be%20renamed%20to%20%3Capp_name%3E%20Admin

Reply:
what if we change this to be 
```suggestion
  config.site_title = 'rails_api_base Admin'
```

so that in `bootstrap.sh` we always search and replace `rails_api_base` for the app name. Maybe we can do that in the whole project instead of searching for specific files (e.g. `package.json` would already benefit from that approach)

Reply:
yeah I like that approach. we just need to check what gets changed but we are probably fine. Let us know how it goes

Reply:
I prefer to defined a list of file types to rename and skip issues with some files like (Gemfiles.lock), also I created a backup of each file that then will deleted, this is because it's possible to get an error with reserve files locals in node_modules